packages/connection-info/src/index.ts

```diff
@@ -3,5 +3,6 @@ export { extractSecrets, mergeSecrets } from './connection-secrets';
 export type {
   ConnectionInfo,
   ConnectionFavoriteOptions,
+  ConnectionStatus,
```

Comment:
leftover? it is not used in the file? or at-least no changes related to this?

Reply:
It's an export in the index.ts file, this will be used by the ConnectionManager and the sidebar, but it's still not implemented.

Reply:
Ahh, did not realise. Thank you :)